core/src/main/java/com/graphhopper/routing/util/parsers/MountainBikePriorityParser.java

```diff
@@ -32,18 +29,12 @@ protected MountainBikePriorityParser(DecimalEncodedValue speedEnc, DecimalEncode
     }
 
     @Override
-    void collect(ReaderWay way, double wayTypeSpeed, boolean bikeDesignated, TreeMap<Double, PriorityCode> weightToPrioMap) {
-        super.collect(way, wayTypeSpeed, bikeDesignated, weightToPrioMap);
-
-        String highway = way.getTag("highway");
-        if ("track".equals(highway)) {
-            String trackType = way.getTag("tracktype");
-            if ("grade1".equals(trackType) || goodSurface.contains(way.getTag("surface","")))
-                weightToPrioMap.put(50d, SLIGHT_PREFER);
-            else if (trackType == null)
-                weightToPrioMap.put(90d, PREFER);
-            else if (trackType.startsWith("grade"))
-                weightToPrioMap.put(100d, VERY_NICE);
-        }
+    double collect(ReaderWay way, double wayTypeSpeed, boolean bikeDesignated) {
+        double prio = super.collect(way, wayTypeSpeed, bikeDesignated);
+
+        // worse grade should not mean we prefer it because it is also less fun
```

Comment:
We still should prefer tracktype=grade2, grade3, grade4 and eventually also grade5 more compared to grade1. Imagine a tracktype grade1 and parallel a highway=track with one of the other grades. From a MTB profile I would expect that the tracktype=grade1 is avoided in such a case.

Reply:
> Mountain biking is more of a leisure activity than a means of transportation

We need to differentiate between the A-to-B routing and the planning where we find the spots and then create a nice tour with multiple via points (or a round tour) to get there. IMO the scope of the GraphHopper routing engine here can be surely be "prefer tracks and mtb-alike roads" (or for kurviger "prefer curvy roads and avoid motorways") but our aim should not be to find "nice mtb spots" on the way to our goal and do big detours because of that and accept really ugly surface (it is not fun!), because there is just no good data for this in OSM (except maybe mtb:scale and scenic).